create-pkg-and-metadata.qmd

```diff
@@ -127,10 +127,29 @@ Description: Provides functions for learning about your R libraries, and the
 check()
 ```
 
-### Use GitHub
+### Use GitLab
 
-```{r}
-use_github()
+-   Make an empty repo on GitLab: https://docs.gitlab.com/ee/user/project/
+-   Choose `empty project`
+-   Choose Public or Private
+-   Uncheck "Initialize repository with a README"
+-   Click "Create Project"
+-   Scroll down to "Push an existing folder" section
+-   Find the line that says `git remote add origin https:://gitlab...` and copy it to your clipboard.
+-   Navigate to the Terminal in RStudio and paste the following commands
+
+#### Tell your git repository where its corresponding repository on GitLab is
+
+```{bash}
+git remote add origin https://gitlab.com/boshek/foo.git
```

Comment:
Yeah I considered that but because this is the command the gitlab gives you in their ui and also because I think we ultimately do have to go to the terminal (see other comment) then I'm inclined to stick with this.